File: Python/underautomation/universal_robots/rtde/rtde_output_setup.py
import typing
from underautomation.universal_robots.rtde.internal.rtde_setup_2 import RtdeSetup2
from underautomation.universal_robots.rtde.rtde_output_setup_item import RtdeOutputSetupItem
from underautomation.universal_robots.rtde.rtde_output_data import RtdeOutputData
import clr
import os
clr.AddReference(os.path.realpath(os.path.join(os.path.dirname(__file__), "..",  'lib', 'UnderAutomation.UniversalRobots.dll')))
from UnderAutomation.UniversalRobots.Rtde import RtdeOutputSetup as rtde_output_setup

class RtdeOutputSetup(RtdeSetup2[RtdeOutputSetupItem, RtdeOutputData]):
	def __init__(self, _internal = 0):
		if(_internal == 0):
			self._instance = rtde_output_setup()
		else:
			self._instance = _internal

File: UnderAutomation.UniversalRobots.ObfuscatedSources/Rtde/RtdeOutputSetup.cs
//              WARNING
// This file is an empty shell containing only public C# items.
// The internal code is hidden; to access it, you need to obtain a Source licence of the library.

using Rtde.Internal;
using System.Collections.Generic;
using System.Collections;

namespace Rtde {
	/// <summary>
	/// Defines the set of RTDE output variables (robot-to-client) to subscribe to as a recipe. The <xref href="UnderAutomation.UniversalRobots.Rtde.RtdeOutputData.Timestamp" data-throw-if-not-resolved="false"></xref> variable is added by default.
	/// </summary>
	public class RtdeOutputSetup : RtdeSetup<RtdeOutputSetupItem, RtdeOutputData>, IList<RtdeOutputSetupItem>, ICollection<RtdeOutputSetupItem>, IList, ICollection, IReadOnlyList<RtdeOutputSetupItem>, IReadOnlyCollection<RtdeOutputSetupItem>, IEnumerable<RtdeOutputSetupItem>, IEnumerable {

		/// <summary>
		/// Initializes a new instance with the default <xref href="UnderAutomation.UniversalRobots.Rtde.RtdeOutputData.Timestamp" data-throw-if-not-resolved="false"></xref> variable.
		/// </summary>
		public RtdeOutputSetup()
		{
			// Source is hidden, a Source licence is needed to access internal code...
		}


		public override string ToString()
		{
			// Source is hidden, a Source licence is needed to access internal code...
			return default;
		}
	}
}
